app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
```

Comment:
- [ ] [OPTIONAL] I think, it could be better to add an `input` for `bio` when we are signing up, to make sure we get data for bio but not `Developer` as you put as the default value.

![Capture](https://github.com/Marcraphael12/rails-blog-app/assets/57408419/c74ffab0-7a4c-42b3-bfdf-5f89f394cddc)


- [ ] [OPTIONAL] Please make sure you modify the views in devise for registration and login to match the wireframes from the Sneak Peek and your styling as it is [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Modify%20the%20views%20in%20devise%20for%20registration%20and%20login%20to%20match%20the%20wireframes%20from%20the%20Sneak%20Peek%20and%20your%20styling.)

Given:

![281605524-00016314-8807-4af2-990b-7f058528efc5](https://github.com/Marcraphael12/rails-blog-app/assets/57408419/ac8fe619-20d1-41aa-95d6-ff7eab921622)

Your project

![Capture](https://github.com/Marcraphael12/rails-blog-app/assets/57408419/e304413c-4377-4f34-82e4-b93d9c8264de)
![Capture](https://github.com/Marcraphael12/rails-blog-app/assets/57408419/717f305d-a6bf-412f-9c1f-7cdf772cff54)

Reply:
Keep doing good job @Marcraphael12